src/InvertedIndices.jl

```diff
@@ -173,4 +199,8 @@ end
     I.skip ⊆ keys(nt) ? Base.structdiff(nt, NamedTuple{I.skip}) :
                         error("type NamedTuple has no fields $(join(I.skip, ", "))")
 
+@inline Base.to_indices(A, inds, I::Tuple{InvertedIndex{GeneralNotMultiIndex}, Vararg{Any}}) =
+    throw(ArgumentError("Conversion of other multiple arguments than integers " *
```

Comment:
```suggestion
    throw(ArgumentError("Conversion of multiple arguments other than integers " *
```
I'm not sure "conversion" is clear for users, as typically they won't have requested a conversion directly. This happens when trying to index an array, right?

Reply:
OK - I have simplified the wording here. This case happens when we pass multiple positional arguments to `Not` and not all of them are integers, e.g. `Not(:a, :b)` and the object we want to index with such a value does not explicitly handle such indexing (in practice - except for DataFrames.jl after we add this support no other object currently will handle this).